demo/src/app/components/typeahead/overview/sections/user-interaction/template.html

```diff
@@ -0,0 +1,82 @@
+<h5>Edit the search term</h5>
+
+<p>
+  The classic usage of the Typeahead is to use the input to edit the search term, and trigger a search process which will spawn a new options list.
+</p>
+
+<p>
+  Apart from creating an Observable based on the <code>input</code> event of the input, the Typeahead does nothing else specific, so to edit the search term just use it as any other input.
+</p>
+
+
+
+<h5>Navigate through options in the popup</h5>
+
+<table class="table mt-4">
+  <tbody>
+    <tr>
+      <td><kbd>ArrowUp</kbd></td>
+      <td>Activate previous (above) option</td>
```

Comment:
See comment below

Reply:
ok, agreed. I just find `(below)` and `(above)` redundant I guess

Reply:
Oh that I can remove. Luckily with a vertical layout I guess we don't have the same issues as for "trimStart" vs "trimLeft", etc. due to RTL mode. So "previous" and "above" will always remain equivalent in this case and same for "next" and "below". I'll keep "below" and "above" since this is directional, like the arrow keys are.